### PR TITLE
fix: disable auto-commit by default

### PR DESCRIPTION
**Pull request by loicmathieu**
Disable kafka-autocommit by default to avoid committing offset if the tas fail.

### CODE REVIEW

File: src/main/java/io/kestra/plugin/kafka/AbstractKafkaConnection.java [hunk 1]
```diff
@@ -45,6 +46,11 @@ protected static Properties createProperties(Map<String, String> mapProperties,
                 }
             }));
 
+        if (!properties.contains(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG)) {
```
Comment (tchiotludo) [inline src/main/java/io/kestra/plugin/kafka/AbstractKafkaConnection.java:49]: Must be done only for when using a consumer group

**Review by tchiotludo — CHANGES_REQUESTED** [review]

File: src/main/java/io/kestra/plugin/kafka/AbstractKafkaConnection.java (hunk 1 above)
Comment (loicmathieu) [reply src/main/java/io/kestra/plugin/kafka/AbstractKafkaConnection.java:49]: When no consumer group, this is already the default as there will not be any auto commit (or more precisely, this setting is ignored as the offsets must be handled manually)

Comment (tchiotludo) [reply src/main/java/io/kestra/plugin/kafka/AbstractKafkaConnection.java:49]: yes I know, but explicit is good and more readable

Comment (loicmathieu) [reply src/main/java/io/kestra/plugin/kafka/AbstractKafkaConnection.java]: @tchiotludo I changed as per your request.

**Pull request merged**